Makefile

```diff
@@ -128,6 +146,12 @@ coverage: test
 	cat $(COVERAGE_FILE) | grep -v "_mock.go" > $(COVERAGE_FILE).no-mocks
 	go tool cover -func=$(COVERAGE_FILE).no-mocks
 
+cov-report: gcov2lcov test ## Build test coverage report in lcov format
+	$(GCOV2LCOV) -infile coverage.out -outfile lcov.info
```

Comment:
nit: We should add `lcov.info` to the `.gitignore`

Reply:
done